American Whitewater/ViewControllers/Reaches/RiverFlowControllers/AddRiverFlowTableViewController.swift

```diff
@@ -25,19 +26,57 @@ class AddRiverFlowTableViewController: UITableViewController {
         
     @IBOutlet weak var submitFlowButton: UIButton!
     
-    //let reportingOptions = ["Low", "Low Runnable", "Runnable", "High Runnable", "Too High"]
-    let reportingOptions = [ ["Low":-1], ["Low Runnable":0.1], ["Runnable": 0.45], ["High Runnable": 0.8], ["Too High":1.5] ]
+    // AWTODO: this should be part of models
+    enum FlowOptions: CaseIterable, Equatable {
+        case low, lowRunnable, runnable, highRunnable, tooHigh
+        
+        var title: String {
+            switch self {
+            case .low: return "Low"
+            case .lowRunnable: return "Low Runnable"
+            case .runnable: return "Runnable"
+            case .highRunnable: return "High Runnable"
+            case .tooHigh: return "Too High"
+            }
+        }
+        
+        var value: Double {
+            switch self {
+            case .low: return -1
+            case .lowRunnable: return 0.1
+            case .runnable: return 0.45
+            case .highRunnable: return 0.8
+            case .tooHigh: return 1.5
+            }
+        }
+    }
     
     var unitOptions = ["cfs", "ft", "in"]
     var availableMetrics = [String:String]()
     
     let dateFormatter = DateFormatter()
     var awImagePicker: AWImagePicker!
     
-    var flowObservationValue: Double? = nil
+    // AWTODO: This is the state for the storyboard placeholder that was being shown by default. Is it the correct default?
```

Comment:
Hmmm, maybe it should show a nil state to make sure the user enters something here? We don't want things defaulting to runnable if they aren't.

Reply:
Makes sense -- done